docs/reference/document_tree.rst

```diff
@@ -220,7 +220,7 @@ like this:
     .. code-block:: jinja
 
         {% render(controller(
-            'sonata.admin.doctrine_phpcr.tree_controller:treeAction',
+            'sonata.admin.doctrine_phpcr.tree_controller::treeAction',
```

Comment:
afaik this only works with newer symfony versions. maybe add a note after that block that on symfony < 3.4 people should use single `:`?